README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/lalithsuresh/hiero.svg?branch=master)](https://travis-ci.org/lalithsuresh/hiero)
```

Comment:
No, that URL shows the build status of my fork of the hiero repo. We need to show the status for the main development repo, which is yours.